CondFormats/GeometryObjects/src/classes_def.xml

```diff
@@ -167,14 +167,30 @@
    <field name="copiesInLayers_"  mapping="blob"/>
  </class>
  <class name="HcalSimulationParameters" class_version="0"> 
```

Comment:
shouldn't it be a new version?

Reply:
Changed

Reply:
FYI, a class_version == 0 tells ROOT not to store that object.

Reply:
Should I change it back to 0 then?